controller/workitem.go

```diff
@@ -256,10 +258,32 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 				return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("space", "string").Expected("valid space ID"))
 			}
 		}
+
+		if ctx.Payload.Data.Relationships.Area == nil || ctx.Payload.Data.Relationships.Area.Data == nil || ctx.Payload.Data.Relationships.Area.Data.ID == nil {
+			// The responsiblity of the code below is to check
+			// if the area information is absent or undefined ,
+			// the code should add the work item to the root area.
+
+			// The parent of path of root area is an empty string since it has no parent.
+			parentPathOfRootArea := path.Path{}
+			rootArea, areaLoadErr := appl.Areas().Query(area.FilterBySpaceID(spaceID), area.FilterByPath(parentPathOfRootArea))
```

Comment:
I am not sure how many queries it will fire (I haven't studied that yet). But would it be good to have a method on repository like - `RootArea(spaceID uuid.UUID) *area.area` that returns root area of that space.
similar to [this](https://github.com/almighty/almighty-core/blob/master/iteration/iteration.go#L131)